crates/flowctl/src/auth/mod.rs

```diff
@@ -57,9 +69,11 @@ pub struct Develop {
 impl Auth {
     pub async fn run(&self, ctx: &mut crate::CliContext) -> Result<(), anyhow::Error> {
         match &self.cmd {
-            Command::Token(Token { token }) => {
-                ctx.config_mut().api = Some(config::API::managed(token.clone()));
-                println!("Configured access token.");
+            Command::ServiceAccount(ServiceAccountArgs { path }) => {
+                let sa_file = std::fs::File::open(path)?;
+                let sa: ServiceAccount = serde_json::from_reader(sa_file)?;
+                ctx.config_mut().api = Some(config::API::managed(sa));
+                println!("Configured service account.");
```

Comment:
nit: let the user know they don't need to keep the file around? They might think they must otherwise.